state/validators.md

```diff
@@ -0,0 +1,38 @@
+# Validator Selection Scheme
+
+## Abstract
+
+This document elucidates the validator selection mechanism as outlined in the [Rollkit repository](https://github.com/rollkit/rollkit). The scheme establishes an initial set of validators at the genesis of the blockchain, then leverages a priority-based system to determine the proposer of new blocks, ensuring a decentralized and fair block proposal mechanism.
```

Comment:
The code links should refer to more precise code paths instead of referring to the entire repository.

Reply:
Updated.